### PR TITLE
Create ZkServerWrapper to speed up test.

### PR DESCRIPTION
**Pull request by zzmao**
Create ZkServerWrapper to replace ZKServer. 
In Mac, NetworkUtil.getLocalHostNames() in ZKServer is time-consuming.

### CODE REVIEW

**Comment by zzmao** [comment]
Code of `ZkServerWrapper` is copied from ZKServer `startSingleZkServer` and `shutdown`.

File: ambry-utils/src/test/java/com.github.ambry.utils/TestUtils.java [hunk 1]
```diff
@@ -223,6 +229,60 @@ public static String getRandomKey(int size) {
     return sb.toString();
   }
 
+  /**
+   * A wrapper class to start and shutdown {@link ZooKeeperServer}.
```
[the remark this answers is not in the document]
Comment (zzmao) [reply ambry-utils/src/test/java/com.github.ambry.utils/TestUtils.java]: fixed.

File: ambry-utils/src/test/java/com.github.ambry.utils/TestUtils.java [hunk 2]
```diff
@@ -223,6 +229,62 @@ public static String getRandomKey(int size) {
...
+   * We maintain this class because {@link org.I0Itec.zkclient.ZkServer} calls NetworkUtil.getLocalHostNames()
+   * which takes time in Max OS.
```
Comment (cgtz) [inline ambry-utils/src/test/java/com.github.ambry.utils/TestUtils.java]: minor: s/Max/Mac

File: ambry-utils/src/test/java/com.github.ambry.utils/TestUtils.java [hunk 3]
```diff
@@ -223,6 +229,62 @@ public static String getRandomKey(int size) {
     return sb.toString();
   }
 
+  /**
+   * A wrapper class to start and shutdown {@link ZooKeeperServer}.
+   * We maintain this class because {@link org.I0Itec.zkclient.ZkServer} calls NetworkUtil.getLocalHostNames()
+   * which takes time in Mac OS.
```
Comment (pnarayanan) [inline ambry-utils/src/test/java/com.github.ambry.utils/TestUtils.java]: and that it is unnecessary for our tests?

Comment (zzmao) [reply ambry-utils/src/test/java/com.github.ambry.utils/TestUtils.java]: ZKServer calls NetworkUtil.getLocalHostNames() to
1. logging
2. make sure `"localhost"` is in the list of `NetworkUtil.getLocalHostNames()`. If not, it does nothing(doesn't start server).

In test, I think we are good to go without above.

Comment (pnarayanan) [reply ambry-utils/src/test/java/com.github.ambry.utils/TestUtils.java]: makes sense - might as well add this whole reasoning in the comment.

**Pull request merged**